filament/src/TypedUniformBuffer.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2015 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef TNT_FILAMENT_TYPEDUNIFORMBUFFER_H
+#define TNT_FILAMENT_TYPEDUNIFORMBUFFER_H
+
+#include "private/backend/DriverApi.h"
+
+#include <utils/compiler.h>
+#include <utils/debug.h>
```

Comment:
I don't see panic or assert_invariant in this file, not sure if this include is needed.

For new files I usually put 2021 for the copyright notice.

Reply:
good points